dimod/discrete/discrete_quadratic_model.py

```diff
@@ -279,12 +291,18 @@ def _from_file_numpy(cls, file_like):
 
         data = np.load(file_like)
 
+        try:
+            offset = data['offset']
```

Comment:
could just do `data.get('offset', 0)`

Reply:
Thanks, didn't think to check for a get method in that object

Reply:
I am only speculating that it has one, but worth checking :slightly_smiling_face:

Reply:
Yep, there is one.